query

```diff
@@ -19,4 +19,4 @@ fi
 
 
 #exec java -cp "${jar}" "${app_class}" $@
-exec ./sbt "run-main tech.sourced.gemini.QueryApp $@"
\ No newline at end of file
+exec ./sbt "run-main tech.sourced.gemini.QueryApp $*"
```

Comment:
In POSIX world, it's generally a good idea to always have a newline char at the end of any text file.

Reply:
I agree. But there was no newline, so I just kept it as is. Will add